File: core/src/net/artux/pda/map/controller/notification/NotificationType.java
package net.artux.pda.map.controller.notification;

public enum NotificationType {
    ATTENTION("textures/icons/notification/alert.png", "audio/sounds/pda/pda_news.ogg");

    private final String icon;
    private final String sound;

    NotificationType(String icon, String sound) {
        this.icon = icon;
        this.sound = sound;
    }

    public String getIcon() {
        return icon;
    }

    public String getSound() {
        return sound;
    }
}

File: model/src/main/java/net/artux/pda/model/quest/NotificationType.kt
package net.artux.pda.model.quest

enum class NotificationType(
    val iconId: String
) {
    MESSAGE("textures/icons/notification/message.png"),
    ALERT("textures/icons/notification/alert.png"),
    KILL("textures/icons/notification/kill.png"),
    SEARCH("textures/icons/notification/search.png"),
    MAP("textures/icons/notification/map.png");


}